docs/digger/installation/using-http-proxy.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[using-http-proxy]]
+== Using HTTP/HTTPS Proxy with Aerogear Digger (optional)
+
+WARNING: HTTP Proxies that require authentication are not supported.
+Please read the link:known_issues[Known Issues] for more details.
+
+You can install AeroGear Digger behind proxy by modifying the variables in Ansible playbook.
+
+The installer will automatically configure the following components to use the proxy (if the component requires an outbound Internet access):
+
+* The Jenkins server
+* The Android SDK pod used to download Android SDKs
+* Container/node that will run the build jobs, by adding the environment variables such as `HTTP_PROXY` and `HTTPS_PROXY` in the inventory file. 
+https://github.com/aerogear/digger-installer/blob/master/inventory-sample[Sample Ansible Inventory file].
+* macOS tools for iOS builds. See <<macos-prereqs>>.
+
+=== Ansible Variables for HTTP/HTTPS Proxy
+
+Set these variables in an Inventory file, or via CLI.
+
+For more details, please refer to http://docs.ansible.com/ansible/playbooks_variables.html[Ansible Playbook Variables Docs].
+
+|===
+| Variable name | Description
+| proxy_url
+| This URL is pasted into `http_proxy` and `https_proxy` environment variables.
```

Comment:
# For proxy_url the syntax should be:
# http://<proxy-host>:<proxy-port>
# or
# http://<proxy_user>:<proxy_pass>@<proxy-host>:<proxy-port>

Reply:
resolved

Reply:
only `http...` or `https` as well? @laurafitzgerald

Reply:
@pmacko1 only http

Reply:
resolved